Apps/Src/ReadCarCAN.c

```diff
@@ -68,27 +69,24 @@ static void updateSaturation(int8_t chargeMessage){
     chargeMsgSaturation = newSaturation;
 }
 
+// exception struct callback for charging disable, kills contactors and turns of display
+static void callback_disableContactors(void){
+    // Kill contactors 
+    Contactors_Set(ARRAY_CONTACTOR, OFF, true);
+    Contactors_Set(ARRAY_PRECHARGE, OFF, true);
```

Comment:
Also, just realized that Contactors uses a mutex. In this case, we should probably directly write to the BSP pin in the context of any error handling (especially if we're locking the scheduler!)

Reply:
Thanks, I didn't realize MOTOR_CONTACTOR and ARRAY_PRECHARGE were our two contactors- I thought ARRAY_PRECHARGE was no longer in our control and that it was just MOTOR_CONTACTOR and ARRAY_CONTACTOR.  Is the precharge contactor now the only way to connect to the array, and do we still control it?

Reply:
Also, does this mean that all the mentions of ARRAY_CONTACTOR in ReadCarCAN (which is every contactor call) needs to be changed to ARRAY_PRECHARGE?

Reply:
As for the issue with the mutex/blocking call in the handler, that gets fixed in ReadCarCAN-Restructure, so it should be good once that's merged in.

Reply:
Technically, the ARRAY CONTACTOR is the one no longer in our control. So yes, everything should call ARRAY_PRECHARGE. I have left a comment on the ReadCarCAN issue ticket to fix it, so let's actually not worry about it here as it may get more confusing.

Sounds good on the mutex call. How was that handled; was the mutex just removed?

Reply:
For the mutex call, Contactors_Set was just changed to a direct write to the BSP pin